assets/app/scripts/directives/util.js

```diff
@@ -75,11 +75,11 @@ angular.module('openshiftConsole')
         tag: '=?'
       },
       controller: function($scope, annotationFilter, iconClassFilter) {
-        var icon = $scope.icon = annotationFilter($scope.resource, $scope.tag ? $scope.tag + ".icon" : "icon");
```

Comment:
create a new filter named `tagAnnotationFilter`
if `$scope.tag` is defined, call that, otherwise, call `annotationFilter`